docs/api-aggregation-setup.md

```diff
@@ -0,0 +1,92 @@
+
+# 
+
+1. make sure `# kubectl api-versions` lists `apiregistration.k8s.io/v1beta1`
+
+# this won't show up if you are talking to the insecure port. the insecure port is not behind the aggregator.
+
+# Get the appropriate tls ca/cert/keys from kube
+# TODO how do I know which ones to get? Are they ones given to specific flags on the apiserver, etc etc?
+```
+export SERVING_NAME=server-ca
+export SERVINGCA_CERT=${SERVING_NAME}.crt
+export SERVINGCA_KEY=${SERVING_NAME}.key
+cp /var/run/kubernetes/${SERVINGCA_KEY} .
+cp /var/run/kubernetes/${SERVINGCA_CERT} .
+```
+
+# TODO explain what we're setting up 
+export PURPOSE=server
+echo '{"signing":{"default":{"expiry":"43800h","usages":["signing","key encipherment","'${PURPOSE}'"]}}}' > "${PURPOSE}-ca-config.json"
+
+# TODO generalize this and link with helm config
+```
+export NAME_SPACE=catalog
+export SERVICE_NAME=catalog-catalog-apiserver
+export ALT_NAMES="\"${SERVICE_NAME}.${NAME_SPACE}\",\"${SERVICE_NAME}.${NAME_SPACE}.svc"\"
+echo '{"CN":"'${SERVICE_NAME}'","hosts":['${ALT_NAMES}'],"key":{"algo":"rsa","size":2048}}' | cfssl gencert -ca=${SERVINGCA_CERT} -ca-key=${SERVINGCA_KEY} -config=server-ca-config.json - | cfssljson -bare apiserver
```

Comment:
make sense, just checking. I wish there was a sort of 'bundled form' but that's probably bad practice somehow because you want to expose one and not the other. Better that they are different files.